src/external/persona.ts

```diff
@@ -0,0 +1,225 @@
+/* eslint-disable complexity */
+/* eslint-disable no-console */
+import { parseString } from '@fast-csv/parse';
+import validator from 'validator';
+
+import { personaURL } from '../config/env';
+import UserModel from '../db/models/User';
+import { LINKEDIN_REGEX } from '../utils/constants';
+import { SubscriptionStatus } from '../utils/newsletter';
+
+export type Persona = {
+    id: string;
+    isPublic: boolean;
+    isActive: boolean;
+    title: string;
+    firstName: string;
+    lastName: string;
+    role: string[];
+    loginEmail: string;
+    institution: string;
+    department: string;
+    jobTitle: string;
+    addressLine1: string;
+    addressLine2: string;
+    city: string;
+    state: string;
+    country: string;
+    phone: string;
+    institutionalEmail: string;
+    eraCommonsID: string;
+    website: string;
+    twitter: string;
+    orchid: string;
+    linkedin: string;
+    googleScholarId: string;
+    github: string;
+    facebook: string;
+    acceptedTerms: boolean;
+    acceptedNihOptIn: boolean;
+    acceptedKfOptIn: boolean;
+    acceptedDatasetSub: boolean;
+    interests: string[];
+    egoId: string;
+    story: string;
+    bio: string;
+};
+
+export const validateUniqPersonas = (personas: Persona[]): boolean => {
+    const result = new Map();
```

Comment:
why not simply use an object?
const result = personnas.reduce((xs, x) => ..., {})